src/styles/page.scss

```diff
@@ -0,0 +1,157 @@
+.page {
+  font-family: "Inter", sans-serif;
+
+  background-color: #f7f7f7;
+  color: $mainTextColor;
+  font-weight: 700;
```

Comment:
why 700?
mb will be better to remove this value?
Bc you use 8 times font-weight: 400; in your styles